index.bs

```diff
@@ -1736,6 +2079,230 @@ The <dfn method for=StorageManager>getDirectory()</dfn> method steps are:
 
 </div>
 
+# Observing file system change events # {#observing-the-file-system}
+
+## The {{FileSystemObserver}} interface ## {#api-filesystemobserver}
+
+<xmp class=idl>
+dictionary FileSystemObserverObserveOptions {
+  boolean recursive = false;
```

Comment:
If a folder being observed is removed, which callback will be called?
One for the folder, or one per each file in the folder, or depending on the OS?